hpx/components/containers/partitioned_vector/partitioned_vector_view.hpp

```diff
@@ -182,7 +176,7 @@ namespace hpx
         iterator end()
```

Comment:
It implies to duplicate some code, I think similarly to `partitioned_vector`  _segmented iterators_.